harbors/admin.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.gis import admin
+from parler.admin import TranslatableAdmin
+
+from .models import BoatType, Harbor
+
+
+class BoatTypeAdmin(TranslatableAdmin):
+    pass
+
+
+class HarborAdmin(TranslatableAdmin, admin.OSMGeoAdmin):
+    pass
+
+
+admin.site.register(BoatType, BoatTypeAdmin)
```

Comment:
This is of course fine as well, but just in case you didn't know, there is also handy `register` decorator, ie. 
```
@admin.register(BoatType)
class BoatTypeAdmin(TranslatableAdmin):
    pass
```

Reply:
Yeah, knew about that one, not sure, which approach looks cleaner yet 😄

Reply:
Ok, fair enough :D